src/lightning_lite/utilities/types.py

```diff
@@ -80,9 +80,19 @@ def step(self, metrics: Union[float, int, Tensor], epoch: Optional[int] = None)
 
 
 @runtime_checkable
-class Steppable(Protocol):
```

Comment:
I suggest we keep Steppable. We can use inheritance for protocols

Reply:
done